packages/jasmine/src/jasmine_node_test.bzl

```diff
@@ -51,7 +55,13 @@ def jasmine_node_test(
         tags = tags,
     )
 
+    # append on any user args to the end
+    args = ["--coverage=%s" % coverage] + args
```

Comment:
You should be able to just add this argument to `templates_args` and avoid the `args` altogether. Also, looking for `--coverage=True` seems pretty brittle as a user could set it to `1`. This would be better:

```
templated_args = ["$(location :%s_devmode_srcs.MF)" % name]
if coverage:
  templated_args = templated_args + ["--coverage"]
else:
  templated_args = templated_args + ["--nocoverage"]
```

and then just look for an argument matching `--coverage` in the `jasmine_runner.js`